revhubinterface.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <style>
+            .center {
+              margin: auto; 
+              width: 85%;
+              padding: 8px;
+            }
+            .pg {
+                line-height: 1.4;
+                font-size: 18px;
+            }
+            .float-container {
+                width: 90%;
+                margin: auto;
+                display: flex;
+                flex-wrap: wrap;
+                align-items: flex-start;
+
+            }
+            .float-child {
+                margin-left: 2.5%;
+                margin-right: 2.5%;
+                margin-bottom: -170px;
+                margin-top: 2.5%;
+                width: 44%;
+                float: left;
+                border-radius: 8px;
+                border-color: azure;
+                border-style: solid;                
+            }
+            a:link, a:visited {
+                color: aqua;
+            }
+
+            a:hover, a:active {
+                color:red;
+            }
+
+          </style>
+    </head>
+    <Body style="background-color: #323232; color: azure; font-family: Arial, Helvetica, sans-serif; padding: 15x; padding-top: 30px">
+        <title>
+            REV Hub Interface
+        </title>
+        <h1 align="center">
+            REV Hub Interface
+        </h1>
+        <div class="center">
+            <p class="pg"> 
+                REV Hub Interface: manual rev Expansion Hub control with no progrmaing required
```

Comment:
Honestly just steal the flathub stuff for this https://flathub.org/apps/org.unofficialrevport.REVHubInterface